core/src/main/java/org/bitcoinj/coinjoin/CoinJoinBaseSession.java

```diff
@@ -82,19 +82,17 @@ interface CheckTxOutputs {
         ValidInOuts check(TransactionOutput txout);
     }
 
-    protected ValidInOuts isValidInOuts(List<TransactionInput> vin, List<TransactionOutput> vout, PoolMessage messageIDRet, boolean consumeCollateral) {
+    protected ValidInOuts isValidInOuts(List<TransactionInput> vin, List<TransactionOutput> vout) {
 
         HashSet<Script> setScripPubKeys = new HashSet<>();
         final ValidInOuts result = new ValidInOuts();
         result.messageId = MSG_NOERR;
-        if (consumeCollateral)
-            result.consumeCollateral = false;
+        result.consumeCollateral = false;
```

Comment:
consumeCollateral was a pointer to a `bool` in the C++ code to return a result, but in Java that value is returned as part of a `ValidInOuts` object rather than as separate variable.